src/data/set/finite.lean

```diff
@@ -271,16 +271,12 @@ theorem infinite_of_infinite_image (f : α → β) {s : set α} (hs : (f '' s).i
   s.infinite :=
 mt (finite.image f) hs
 
-lemma finite.dependent_image {s : set α} (hs : finite s) {F : Π i ∈ s, β} {t : set β}
-  (H : ∀ y ∈ t, ∃ x (hx : x ∈ s), y = F x hx) : set.finite t :=
+lemma finite.dependent_image {s : set α} (hs : finite s) (F : Π i ∈ s, β) :
+  finite {y : β | ∃ x (hx : x ∈ s), F x hx = y} :=
```

Comment:
Why did you change the order here? I mean, go from `y = F x hx` to `F x hx = y`? Typically, in applications if you `rcase` along this you will get an assumption `h : y = F x hx` along which you can rewrite directly, while if you get `F x hx = y` you will have to rewrite in the reverse direction, which seems less natural.

Reply:
My goal was to match the definition of `set.image` but I can revert this part (or drop the whole PR - it turns out that I won't need it).

Reply:
I don't mind the PR, but I really prefer the other direction.